From 8312398dbc1f0a960c0e64c2faaa756277c0f67c Mon Sep 17 00:00:00 2001
From: Alyssa Ross <hi@alyssa.is>
Date: Fri, 3 Jan 2025 12:21:28 +0100
Subject: [PATCH 3/3] Don't use non-standard ACCESSPERMS macro

This macro is non-standard, and is not defined by e.g. musl libc.
It's just a define for 0777, so just use that instead.

Link: https://github.com/aja-video/libajantv2/pull/42
---
 ajabase/test/main.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ajabase/test/main.cpp b/ajabase/test/main.cpp
index 78196446..e18b10a3 100644
--- a/ajabase/test/main.cpp
+++ b/ajabase/test/main.cpp
@@ -1951,9 +1951,9 @@ TEST_SUITE("file" * doctest::description("functions in ajabase/system/file_io.h"
 				_mkdir(tempDir.c_str());
 				_getcwd(cwdBuf, AJA_MAX_PATH);
 	#else
-				if (mkdir(tempDir.c_str(), ACCESSPERMS) == 0)
+				if (mkdir(tempDir.c_str(), 0777) == 0)
 				{
-					chmod(tempDir.c_str(), ACCESSPERMS);
+					chmod(tempDir.c_str(), 0777);
 				}
 				char* result = getcwd(cwdBuf, AJA_MAX_PATH);
 				AJA_UNUSED(result);
-- 
2.49.0

